### PR TITLE
Update TRANSLATION.md

### PR DESCRIPTION
**Pull request by jdougnac**
translated the page.

 On line 11, I translated "figures" as "images" since I'm not certain of context. If provided with a few synonyms, I might get a more precise wording going. Perhaps "numbers" or "ciphers"?

On line 27, I translated "fork" as "bifurcación".

On line 28, "automatic build server" as "servidor de construcción automático".

I suggest having one more person, who is a native Spanish speaker more versed in github than I am, to double check my translation. I'm doing my best, but some mistakes are inevitable due to my lack of expertise with github and python.

### CODE REVIEW

File: TRANSLATION.md [hunk 1]
```diff
@@ -1,58 +1,61 @@
...
+=============================
+
+Los materiales del curso están licenciados bajo Creative Commons, por lo que pueden usarse
+siempre que se haga la correspondiente atribución. Estaré encantado de ayudar a
+quienes usen o quieran usar estos materiales a comprobar que su atribución
+es suficiente. Siempre recibo gustoso la noticia de nuevas traducciones, las que
+promociono a mis estudiantes.
+
+Todo el material de este curso (sitio web, autoevaluadores, presentaciones PowerPoint,
+libros de texto, imágenes del mismo, tareas, etc) se encuentra en este repositorio de GitHub. El
```
Comment (don-charlie-pc) [inline TRANSLATION.md:11]: In my opinion, using "imágenes" for "figures" sounds good.

File: TRANSLATION.md [hunk 2]
```diff
@@ -1,58 +1,61 @@
...
-github and produces the PDF and EPUB and hosts it online at 
+Mi meta final sería una serie de sitios web completos en urls como es.py4e.com, 
+cn.py4e.com o ir.pye4.com con experiencias de usuario completamente traducidas que
+además funcionen como repositorios OER para los materiales de Python en esos lenguajes.
+Evidentemente, no necesitas usar mis urls, pero quiero promover la traducción de estos 
+materiales y estaré encantado de apuntar mis subdominios a tus servidores o reenviar
+solicitudes a estos subdominios a tus servidores.
+
+He configurado mi repositorio de github de forma tal que puedas hacer una 
+bifurcación de éste y comenzar a traducirlo. Estaré encantado de subir y poner
```
Comment (don-charlie-pc) [inline TRANSLATION.md]: I am not sure how other Spanish speakers talk about a fork. In the case of Mexico, the term "fork" is used as is, but I am not sure about other countries. @jdougnac have you heard of the translation of the word in another countries?

Comment (don-charlie-pc) [reply TRANSLATION.md]: Maybe it would be clearer if we say something like "...tal que puedas hacer una copia (fork) de éste y comenzar...".

Comment (jdougnac) [reply TRANSLATION.md]: efectuada la modificación

**Review by don-charlie-pc — APPROVED** [review]

**Pull request merged**